src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,33 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
```

Comment:
The `Session` object is initialized to `null`. Make sure that the `factory` field in the `AbstractDao` class is properly initialized to avoid `NullPointerException` when calling `factory.openSession()`.